plugin/Record/RecordManagerReadForm.php

```diff
@@ -206,17 +229,21 @@ public function getRowByName($formName)
 	 *
 	 */
 
-	public function readFieldconfigByFormId($formId)
+	public function readFieldconfigByFormId(int $formId)
 	{
 		$prefix = $this->getTablePrefix();
 		$pWpDb = $this->getWpdb();
 
-		$sqlFields = "SELECT *
-			FROM {$prefix}oo_plugin_form_fieldconfig
-			WHERE `".esc_sql($this->getIdColumnMain())."` = ".esc_sql($formId)."
-			ORDER BY `order` ASC";
+		$sql = $pWpDb->prepare(
+			"SELECT *
+			FROM %i
+			WHERE %i = %s",
```

Comment:
Wenn $formId ein int ist, sollte hier statt %s %d sein, oder?
```suggestion
			WHERE %i = %d",
```